### PR TITLE
Dockerfile for local server

### PR DESCRIPTION
**Pull request by khallnayak**
Created Dockerfile with flexibility for changing port and ip version. #24 

# BUILD

```
docker build -t fedora-easyfix:1.0 .
```

# RUN
## Default port option
```
docker run -it -p 9696:9696 fedora-easyfix:1.0
```

## Different port number

```
docker run -it -p <Port number>:<Port number> fedora-easyfix:1.0 -4 -p <Port number>
```

### GG

### CODE REVIEW

**Comment by khallnayak** [comment]
Please Hacktoberfest accept kar

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-alpine
+
+COPY . /opt/easyfix/.
+
+LABEL maintainer "khallnayak <mahijmomin@gmail.com>"
```
Comment (t0xic0der) [inline Dockerfile:5]:
```suggestion
LABEL maintainer "Mahij Momin <mahijmomin@gmail.com>"
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+WORKDIR /opt/easyfix
+
+RUN apk add git & \
+    pip3 install -r requirements.txt & \
+    python3 setup.py install
```
Comment (t0xic0der) [inline Dockerfile:11]:
```suggestion
RUN apk add git
RUN pip3 install -r requirements.txt
RUN python3 setup.py install
```

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by t0xic0der — APPROVED** [review]
LGTM.

**Pull request merged**